types/ramda/index.d.ts

```diff
@@ -49,6 +49,7 @@ declare namespace R {
     }
 
     type Pred = (...a: any[]) => boolean;
+    type SafePred<T> = (...a: T[]) => boolean;
```

Comment:
Just confirming, is your intention to have a function type whose arguments are all the same type? Because that's the only thing that this allows. e.g.: `(a: string, b: string, c: string) => boolean`

Reply:
Good point. We would need to add multiple overloads to support different parameter types.

Reply:
I think you can achieve a typed `anyPass` by simply using `extends` in the type parameter, without having to declare a new type.

```anyPass<T extends Pred>(preds: Array<T>): T;```